backend/application/syson-sysml-import/pom.xml

```diff
@@ -59,12 +59,7 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-web-services</artifactId>
-			<version>${sirius.web.version}</version>
-		</dependency>
-		<dependency>
-			<groupId>org.eclipse.sirius</groupId>
-			<artifactId>sirius-components-starter</artifactId>
+			<artifactId>sirius-web-application</artifactId>
```

Comment:
Could you please verify that it is really sirius-web-application that should be added here? (I just want to be sure that it is not sirius-web-starter that should be added)

Reply:
No need to depend on the `sirius-web-starter`, in fact there is no need to depend on `sirius-web-application` either, but all dependencies needed by `syson-susml-import` are needed by `sirius-web-application`.
If it's need we could split into smaller dependencies.